### DISCUSSION TITLE
Suricata: Custom rule location

### DISCUSSION THREAD

**Discussion by wildlyunder** [Security-Onion-Solutions/securityonion#7154] [Unsupported Versions] [answered]
Hi,

I'm trying to add the following Suricata

`pass http $HOME_NET any -> $EXTERNAL_NET any (flow:established,to_server; content:"windowsupdate.com"; http_host; isdataat:!1,relative; sid:9000001; rev:1;)`

Where would be the correct place to add this rule?

**Comment by dougburks** [comment, marked as answer]
Please see the documentation:
https://docs.securityonion.net/en/2.3/local-rules.html
